### PR TITLE
Migrate to using Firebase Web SDK

### PR DESCRIPTION
**Pull request by gugahoi**
We are preferring using the SDK directly in favour of adding the scripts
to the DOM. This is due to the fact that loading the script by adding
them to the DOM is unreliable and provides potential sources of
conflicts with other Firebase related Capacitor plugins.

### CODE REVIEW

**Comment by gugahoi** [comment]
This is what I meant when I said we should look to get rid of the scripts in favor of the SDK. If this works the way I expect it to, then we don't need to worry about loading the scripts in HTML and waiting for them to be ready. Let me know your thoughts.

File: README.md [hunk 1]
```diff
@@ -19,10 +19,9 @@
 
 ## Maintainers
 
-| Maintainer    | GitHub                                      | Social                                           |
-| ------------- | ------------------------------------------- | ------------------------------------------------ |
-| Priyank Patel | [priyankpat](https://github.com/priyankpat) | [@priyankpat\_](https://twitter.com/priyankpat_) |
-| Stewan Silva  | [stewwan](https://github.com/stewwan)       | [@StewanSilva](https://twitter.com/StewanSilva)  |
+| Maintainer      | GitHub                                |
+| --------------- | ------------------------------------- |
+| Gustavo Horisch | [gugahoi](https://github.com/gugahoi) |
```
Comment (Nevvulo) [inline README.md:24]: Typo in your last name, no need to fix just thought I'd flag

**Review by Nevvulo — APPROVED** [review]
Code-wise this looks good, I've also tested this PR with the related change in `firebase-remote-config`, tested on Web + Android and experienced no errors

File: README.md (hunk 1 above)
Comment (gugahoi) [reply README.md:24]: 🤦

**Pull request merged**